### PR TITLE
feat/3: Added new config to allow users to toggle total card counters

### LINKED ISSUE #3: Add config to toggle card count
The cards count in the list header is currently always visible when the extension is on.

A new config `cardCountersActive` can be a nice to have feature.

### PR DESCRIPTION
**Pull request by jbarbosa7**
This PR fixes #3 

- Added a new config to allow users to toggle the total card counters labels

### CODE REVIEW

**Review by Balastrong — CHANGES_REQUESTED** [review]
Thank you for the PR, I see the code style of the project is perfectly matched, well done!

I added a little note in the other comment, everything else looks good :)

File: public/popup.html [hunk 1]
```diff
@@ -13,6 +13,15 @@
...
+          <div class="config-label">Card Counters</div>
+          <div class="config-control">
+            <label class="switch">
+              <input type="checkbox" id="card-counters-active" />
+              <span class="slider round"></span>
+            </label>
+          </div>
+        </div>
         <div class="config-row">
           <div class="config-label">Active</div>
```
[the remark this answers is not in the document]
Comment (jbarbosa7) [reply public/popup.html:26]: Great catch! I actually thought about that but then forgot to change it 😓 I'll update this as soon as i can :)

**Comment by jbarbosa7** [comment]
> Thank you for the PR, I see the code style of the project is perfectly matched, well done!
> 
> I added a little note in the other comment, everything else looks good :)

@Balastrong 

> Thank you for the PR, I see the code style of the project is perfectly matched, well done!
> 
> I added a little note in the other comment, everything else looks good :)

Already updated the PR with your sugestion :)

**Review by Balastrong — APPROVED** [review]
Thank you :D

**Pull request merged**